test/integration/ext/openai/test_model_openai.py

```diff
@@ -24,12 +25,30 @@
     mp.setenv('OPENAI_API_KEY', 'sk-TopSecret')
 
 
-def _record_only_png_signature_in_response(response):
+# monkey patch vcr to make it work with upload binary data
+def _make_vcr_request(httpx_request, **kwargs):
+    from vcr.request import Request as VcrRequest
+
+    try:
+        body = httpx_request.read().decode("utf-8")
+    except UnicodeDecodeError:
+        body = str(httpx_request.read())
+    uri = str(httpx_request.url)
+    headers = dict(httpx_request.headers)
+    return VcrRequest(httpx_request.method, uri, body, headers)
```

Comment:
In the latest version, when vcr decodes the uploaded binary file, an error occurs. We need to patch it here.

Reply:
We will get a error on `decode` for the binary data